test/pytest/test_example_scriptable_tokenzier.py

```diff
@@ -318,7 +318,7 @@ def test_inference_with_pretrained_model(model_store, test_file, torchserve):
     assert "Positive" in result_entries
 
     assert float(result_entries["Negative"]) == pytest.approx(
-        0.0001851904089562595, 1e-3
+        0.0001851904089562595, abs=1e-6
```

Comment:
Why not apply this uniformly to 323 as well?

Reply:
Done. Somehow the pytest showed a failure only here